README.md

```diff
@@ -83,11 +83,14 @@ jobs:
     steps:
       # To use this repository's private action, you must check out the repository
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.CUSTOM_GITHUB_PAT }}
```

Comment:
I don't think the former example with `${{ secrets.GITHUB_TOKEN }}` is not working. Maybe I do not fully understand your issue #422.
If the former example is working, maybe we add a further example public with PAT?

Reply:
Thanks for the quick reply @AndreasAugustin ! 
It's the first time that I use this action on a public repository, so maybe I'm missing something...

I will need to make further tests to confirm, but the issue might be that I'm trying to sync a public repository containing GitHub workflows, so the default privileges might not be enough for this special case. On private repositories, I'm always using a PAT with permissions on my workflows & repositories, so it doesn't matter, but it could explain the difference in this case.

Here's the data:

The first run is without any token:
- [Workflow file 62b33332237fedefde8afdc85c7c1705c6fea73b](https://github.com/dotdc/repo-sync-issue/blob/62b33332237fedefde8afdc85c7c1705c6fea73b/.github/workflows/repo-sync.yml)
- [Workflow run #01 log](https://github.com/dotdc/repo-sync-issue/actions/runs/7350774911/job/20013019734) :red_circle: Failed

The second run is with your `${{ secrets.GITHUB_TOKEN }}` suggestion :
- [Workflow file cd0af66d5bf0051a9d30b1206e42b9cf7fc81dba](https://github.com/dotdc/repo-sync-issue/blob/cd0af66d5bf0051a9d30b1206e42b9cf7fc81dba/.github/workflows/repo-sync.yml)
- [Workflow run #02 log](https://github.com/dotdc/repo-sync-issue/actions/runs/7350795204/job/20013077600) :red_circle: Failed

The third one is with my own PAT :
- [Workflow file 0708f3f770afe83eb32aa41e0955cfcc2a90e4bd](https://github.com/dotdc/repo-sync-issue/blob/0708f3f770afe83eb32aa41e0955cfcc2a90e4bd/.github/workflows/repo-sync.yml)
- [Workflow run #03 log](https://github.com/dotdc/repo-sync-issue/actions/runs/7350998740/job/20013615278) :green_circle: Success

If it appears to be an issue with the sync of GitHub workflow files, I can change my PR to better explain this case in the documentation if it's fine for you?

Reply:
Indeed if you need to sync workflow files you are not able to use the GITHUB_TOKEN.  But the error message looks different than the one mentioned within #422 
https://github.com/AndreasAugustin/actions-template-sync#troubleshooting

With PAT this should work but this is not documented (yet).

Reply:
Will make the last tests tomorrow and will update my PR to reflect that in the doc :+1: